ci/jenkins/test-vmc.sh

```diff
@@ -252,20 +252,20 @@ function copy_image {
   IP=$3
   version=$4
   need_cleanup=$5
-  scp -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key $filename capv@${IP}:/home/capv
+  scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key $filename capv@${IP}:/home/capv
```

Comment:
Maybe use a variable to reduce code redundancy, like:
```
SCP="scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key"

${SCP} $filename capv@${IP}:/home/capv
```

Reply:
Good idea. Thanks.